### DISCUSSION TITLE
5/11 백엔드 회의

### DISCUSSION THREAD

**Discussion by jinbekim** [PPiing/PPoong#24] [Backend discussions]
## 안건
1. entity 네이밍
2. eslint 수정건 -> 순환참조
3. 소켓명세 구체화.

**Comment by JAWSP** [comment]
![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/313fe2a3-3de3-4e22-b44a-2312f06cbc08/Untitled.png)

### 진행건에 대하여

- 그냥 되는 사람들끼리 모여서 후딱 진행하는게 더 효율적일듯

### 핑퐁 최적화

- 핑퐁을 구현할 떄 공의 위치를 어찌 서버로 보낼지 문제

### 백앤드 엔터티 만든거 MERGE하기

팀원들 리스펙해

- merge하면서 conflict할때 해결을 함
- 결론은 죽이고픈 깃허브

### ORM설정하기

- 뭔갈..뭔가 많이 설정한 것 같은데
- 이것 저것 필요할 것 같다는걸 구글에서 긁어와서 설정을 넣었는듯하다
- 그리고 CI에서 쿠사리 넣은 부분도 수정을 한듯하다

## 이게 뭔데

- e2e test

[https://hyg4196.tistory.com/127](https://hyg4196.tistory.com/127)

- e2e(End to End)테스트는 개발한 것을 사용자 입장으로 테스트를 해보는것이다
- 화면이 잘뜨나 클릭이 잘되나 이런느낌이라고 한다

### CI가 왜 쿠사리 넣게 되었나

- CI의 lint가 계속 쿠사리를 넣게 되었다고한다

→이상허네

- lint 를 패키지를 지우고 다시 고치니 빌드쪽에서 쿠사리를 넣게 되었다

### 너님 Node 버전 확인하십시오

- 어쩌면 이 일의 원인이 버전이 서로 달라서 이렇게 될 수 있다고 한다

### package.lock.json파일을 고심끝에 삭제

- 메인버전이랑 버전 충돌이 나서 삭제를 했는데
- X망

### 깃허브가 시키는대로 하니까 머지가 되었다

- 와~ 역시 정석이 나은가보네

### ERD랑 엔터티만든거랑 비교해보기

**Comment by joohongpark** [comment]
TODO
 - @joohongpark CI 이슈 해결
 - @jinbekim @muingY @joohongpark @KKWANH @yamkim
   - 오늘 결정된 엔티티 ERDCloud에 확정된것과 비교하기
   - CRUD 및 테스트 적용해보기
 - 온라인 (목요일 저녁 아홉시쯤) 후속 회의

**Reply by jinbekim** [reply]
10시까지 CRUD 적용해야 하는 건가요?

**Reply by joohongpark** [reply]
완성된 결과물 보다는 단순히 CRUD가 어떤식으로 되는지 각자 체험해보고 실제로 어떻게 어떤 방식으로 진행할지를 오늘 회의에서 의논하는 것으로 알고 있어요

**Reply by jinbekim** [reply]
아하아하 넵넵
